src/background.js

```diff
@@ -1762,9 +1779,12 @@ function init() {
 		initializeEventListeners();
 		initializeVersioning();
 		return metrics.init(globals.JUST_INSTALLED).then(() => initializeGhosteryModules().then(() => {
+			ghosteryDebug.init();
+			ghosteryDebug.addAccountEvent('migrate', 'migrate start');
```

Comment:
Per Jose, standardize this by moving inside the migration call.

Reply:
Keeping `init` outside so it won't be dependent on `account.migrate` resolving.